### PR TITLE
[Serializer] Name Converter

### PR DESCRIPTION
**Pull request by dunglas**
| Q | A |
| --- | --- |
| Doc fix? | no |
| New docs? | yes symfony/symfony#13120 |
| Applies to | 2.7 |

This is the documentation for the Serializer Name Converter feature.

### CODE REVIEW

**Comment by WouterJ** [comment]
I think this one can be merged now

File: components/serializer.rst [hunk 1]
```diff
@@ -162,38 +162,117 @@ needs three parameters:
...
+    interface was introduced in Symfony 2.7.
 
-Sometimes property names from the serialized content are underscored (e.g.
-``first_name``).  Normally, these attributes will use get/set methods like
-``getFirst_name``, when ``getFirstName`` method is what you really want. To
-change that behavior use the
-:method:`Symfony\\Component\\Serializer\\Normalizer\\GetSetMethodNormalizer::setCamelizedAttributes`
-method on the normalizer definition::
+Sometimes serialized attributes must be named differently than properties
+or getter / setter methods of PHP classes.
```
Comment (xabbuh) [inline components/serializer.rst]: I think the spaces around the `/` should be removed.

File: components/serializer.rst [hunk 2]
```diff
@@ -162,38 +162,117 @@ needs three parameters:
...
-    $serializer = new Serializer(array($normalizer), array($encoder));
+Given you have the following object::
+
+    class Company
+    {
+        public name;
+        public address;
+    }
+
+
```
Comment (xabbuh) [inline components/serializer.rst]: Please remove one blank line.

File: components/serializer.rst [hunk 3]
```diff
@@ -162,38 +162,117 @@ needs three parameters:
...
+And in the serialized form, all attributes must be prefixed by ``org_`` like
+the following::
+
+    {"org_name": "Acme Inc.", "org_address": "123 Main Street, Big City"}
+
+A custom Name Converter can handle such cases::
```
Comment (xabbuh) [inline components/serializer.rst]: name converter (lowercased)

**Comment by dunglas** [comment]
Issues found by @xabbuh fixed.

**Comment by weaverryan** [comment]
Thanks Kévin! I'm excited to try out all the new serializer features in 2.7 - much needed imo!

**Pull request merged**